examples/C/depth/rs-depth.c

```diff
@@ -50,7 +50,8 @@ float get_depth_unit_value(const rs2_device* const dev)
 
         if (1 == is_depth_sensor_found)
         {
-            depth_scale = rs2_get_option((const rs2_options*)sensor, RS2_OPTION_DEPTH_UNITS, &e);
+            if( rs2_supports_option((const rs2_options*)sensor, RS2_OPTION_DEPTH_UNITS, &e) )
```

Comment:
Don't we have depth scale on D555?

Reply:
The camera does not publish this as an option. You can use the CPP API `depth_sensor.get_depth_scale()`, but this is a C example.